bootstrap/kubeadm/internal/bottlerocket/bottlerocket.go

```diff
@@ -265,6 +273,13 @@ func getBottlerocketNodeUserData(bootstrapContainerUserData []byte, users []boot
 		}
 
 	}
+	if config.CertBundle != nil {
+		for _, cert := range config.CertBundle {
+			cert.Data = base64.StdEncoding.EncodeToString([]byte(cert.Data))
```

Comment:
I thought we were asking users to pass encoded certs in the clusterconfig? Are we expecting them to pass the actual cert now?

Reply:
Yeah, we changed the expectation to users passing in the actual certs now